### DISCUSSION TITLE
I removed the API Gateway from the project, the deploy doesn't work anymore

### DISCUSSION THREAD

**Discussion by theliuk** [mia-platform/community#314] [Q&A] [answered]
### Which section?

Console

### Which is the version?

1.87.2

### Description

I removed the API Gateway from the "Design/Microservice" section and now the deployment doesn't work.
The API Gateway was the one from the Marketplace made with Envoy.

The deploy pipeline is based on `platform/pipelines-templates/deploy/deploy-job-v4.yml`

The error I get is:

```
$ mlp deploy --ensure-namespace=false --server "${KUBE_URL}" --certificate-authority ${KUBE_CA_PEM} --token "${KUBE_TOKEN}" --deploy-type "${DEPLOY_TYPE}" --force-deploy-when-no-semver=${FORCE_DEPLOY_WHEN_NO_SEMVER} -f "${DESTINATION_PATH}/kustomize-output.yaml" -n "${KUBE_NAMESPACE}"
2023/09/18 15:37:46 invalid configuration: unable to read certificate-authority --token for  due to open --token: no such file or directory
```

### Actual Outcome

The deploy pipeline doesn't work

### Expected Outcome

The deploy pipeline works

**Comment by fredmaggiowski** [comment]
The problem does not seem to be related with your action of removing the API Gateway, it seems like a networking error between the pipeline runner and the cluster, does the problem still persist?

**Reply by theliuk** [reply]
When I deploy the branch with the api-gateway it works. Only if I remove the api-gateway it doesn't work. I tried a few moments ago and it didn't work.

**Reply by theliuk** [reply]
You're right, the problem isn't the api-gateway, the problem is deploying a branch that isn't `master`. I branched out from master and I tried to deploy the new branch and it doesn't work (no edits on the new branch).

**Reply by theliuk** [reply]
For my company deploy of branches doesn't work anymore :(

**Reply by fredmaggiowski** [reply, marked as answer]
I'd suggest you to open a SR, it is most likely a configuration issue of your project 🤔

**Reply by theliuk** [reply]
It was a configuration problem, yes.
